tests/unit/test_schemas.py

```diff
@@ -0,0 +1,46 @@
+import os
+import subprocess
+
+import pytest
+from ruamel.yaml import YAML
+
+import tmt
+
+PATH = os.path.dirname(os.path.realpath(__file__))
+SCHEMADIR = os.path.join(PATH, "../../tmt/schemas")
+
+# make sure tmt tree is initialized, required when tests run during rpmbuild
+tmt.base.Tree.init('.', 'empty', False)
+
+
+@pytest.fixture
+def schema_and_store():
+    schema_file = os.path.join(SCHEMADIR, 'tests.yaml')
```

Comment:
Please, add a TODO for me: tmt package shall provide a helper function for "load schemas for test/plan/story" - I will add such a function in my patch, and update the test to use it.

Reply:
```suggestion
    # TODO: tmt package shall provide a helper function
    # for  "load schemas for test/plan/story"
    schema_file = os.path.join(SCHEMADIR, 'tests.yaml')
```

Reply:
@happz added